ibis/backends/pandas/client.py

```diff
@@ -298,6 +299,28 @@ def convert_array_to_series(in_dtype, out_dtype, column):
 sch.Schema.to_pandas = ibis_schema_to_pandas  # type: ignore
 
 
+# ImmutableProxy?
+class DataFrameProxy:
```

Comment:
It should subclass `Immutable` in the `4.x.x` branch.

Reply:
Ahh, we already have immutable here. Could you please subclass it and remove the comment above?

Reply:
Yep

Reply:
Done.